src/sass/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$medium-gray: #636c72;
+$light-gray: #d4d4d4;
+$white: #fff;
```

Comment:
Bootstrap already defines `$white`, so we don't have to define it again.

Reply:
👍

Reply:
oh wait, i don't actually know how to use a bootstrap value in scss....